File: app/src/main/java/com/ajdi/yassin/popularmovies/data/local/model/Genre.java
package com.ajdi.yassin.popularmovies.data.local.model;

import com.google.gson.annotations.SerializedName;

/**
 * @author Yassin Ajdi
 * @since 11/20/2018.
 */
public class Genre {

    @SerializedName("id")
    private int id;

    @SerializedName("name")
    private String name;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: app/src/main/java/com/ajdi/yassin/popularmovies/data/local/model/Genre.kt
package com.ajdi.yassin.popularmovies.data.local.model

import com.google.gson.annotations.SerializedName

/**
 * @author Yassin Ajdi
 * @since 11/20/2018.
 */
data class Genre (

    @SerializedName("id")
    var id: Int = 0,

    @SerializedName("name")
    var name: String? = null
)
